src/duration/mod.rs

```diff
@@ -692,7 +690,15 @@ impl fmt::Display for Duration {
             }
 
             let values = [days, hours, minutes, seconds, milli, us, nano];
-            let units = ["days", "h", "min", "s", "ms", "μs", "ns"];
+            let units = [
+                if days > 1 { "days" } else { "day" },
```

Comment:
Super minor but it was driving me crazy that a duration of one day was printed in plural form.